e2e/test/MessageSendFaultInjectionTests.cs

```diff
@@ -146,78 +137,119 @@ public async Task Message_AmqpD2CLinkDropSendRecovery_AmqpWs()
                 FaultInjection.FaultCloseReason_Boom,
                 FaultInjection.DefaultDelayInSec).ConfigureAwait(false);
         }
-
-        // WIP: Exception handling
+        
         [TestMethod]
         public async Task Message_ThrottledConnectionRecovery_Amqp()
         {
-            await SendMessageRecovery(
+            try
+            {
+                await SendMessageRecovery(
```

Comment:
Why would we expect an error on a recovery test that, from the name, it's supposed to pass.
Please name the tests like _ComponentUnderTest___Action___ExpectedResult__[_Protocol_]